<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZipDialog</class>
 <widget class="QDialog" name="ZipDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="fileNameEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="executeButton">
     <property name="text">
      <string>Execute</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
